engine/access/rpc/backend/backend_scripts.go

```diff
@@ -111,13 +112,10 @@ func (b *backendScripts) executeScriptOnExecutionNode(
 	script []byte,
 	arguments [][]byte,
 ) ([]byte, error) {
-
-	execReq := &execproto.ExecuteScriptAtBlockIDRequest{
-		BlockId:   blockID[:],
-		Script:    script,
-		Arguments: arguments,
+	tryScriptExecFunc := b.tryExecuteScriptOnExecutionNode
```

Comment:
maybe refactor this into a method and just call b.tryExecuteScript below

Reply:
this method (`executeScriptOnExecutionNode`) should also be renamed